pkg/registry/registry_antrea.csv

```diff
@@ -54,3 +54,5 @@ ElementID,Name,Abstract Data Type,Data Type Semantics,Status,Description,Units,R
 152,flowEndSecondsFromDestinationNode,unsigned32,,current,,,,,,,,56506,
 153,egressName,string,,current,,,,,,,,56506,
 154,egressIP,string,,current,,,,,,,,56506,
+155,isL7,string,,current,,,,,,,,56506,
```

Comment:
what is the meaning of `isL7`? I think the format allows for quick description of each IE?

Reply:
Thanks for pointing it out, I have added the meaning of the added fields in the description of the PR.